apps/dapp-console-api/src/testhelpers/auth.ts

```diff
@@ -49,17 +44,17 @@ export const createSignedOutCaller = <T extends AnyRouter>(router: T) => {
   })
 }
 
-/** Sets up the mocks required for authentication. */
-export const configureAuthMocks = (privyClient: PrivyClient) => {
```

Comment:
after re-thinking this, I realized there is no clean way to share these mocks and instead should export a session that will allow a user to pass the `isPrivyAuthed` middleware instead.